flexget/plugins/sites/divxatope.py

```diff
@@ -59,15 +58,18 @@ def search(self, task, entry, config=None):
             log.debug('Divxatope disabled')
             return set()
         log.debug('Search DivxATope')
-        url_search = 'http://www.divxatope.com/buscar/descargas/'
+        #url_search = 'http://www.divxatope.com/buscar/descargas/'
+        url_search = 'http://divxatope1.com/buscar/descargas'
         results = set()
         regex = re.compile("(.+) \(\d\d\d\d\)")
         for search_string in entry.get('search_strings', [entry['title']]):
             query = normalize_unicode(search_string)
-            query = regex.findall(query)[0]
+            if regex.findall(query):
+                query = regex.findall(query)[0]
```

Comment:
This?
`
if regex.search(query) is not None:
    query = regex.search(query).group(1)
`